conduct.md

```diff
@@ -213,6 +224,6 @@ Any decision made by the instructor for the given infraction(s) will be final.
 
 Janyl Jumadinova
 :heavy_exclamation_mark: TODO: Add your name
-Bennett Westfall
-Zackery Devers
-Natnael Kahesay
+
+Bennett Westfall, Zackery Devers, Natnael Kahesay, Kobe Coleman, Adriana Solis, Favour Ojo, Kai'lani Woodward, Zoe White
```

Comment:
Please make sure that all names are next to each other. In this case `Janyl Jumadinova` is on a completely different line. Feel free to put the `TODO` before or after the name